looprpc/client.proto

```diff
@@ -317,8 +317,10 @@ message SwapResponse {
     */
     string htlc_address_p2wsh = 5;
 
+    string htlc_address_p2tr = 6;
+
     // A human-readable message received from the loop server.
-    string server_message = 6;
+    string server_message = 7;
```

Comment:
changing the proto number of `server_message` will break the API for old clients.